### PR TITLE
Fix test suite load

### LINKED ISSUE #10: Fix shared test-suite loading
In this implements, reading of the SIDE file performed at each test-project execution.
So, can't use test-suite before test-project execution.

### PR DESCRIPTION
**Pull request by yaamai**
Fix #10

### CODE REVIEW

File: tests/unit/test_main.py [hunk 1]
```diff
@@ -20,3 +22,51 @@ def test_main_with_glob_no_match(mocker):
...
+    sidefile_b = tmp_path / "b.json"
+    orig_test_project_b = {
+        'suites': [{'tests': ['foobar_b']}],
+        'tests': [{'id': 'foobar_b', 'commands': []}], 'id': 'foobar_b'
+    }
+    sidefile_b.write_text(json.dumps(orig_test_project_b))
+
+    # call main with mocked driver, etc...
+    with mock.patch.object(config.sys, 'argv', ['prog_name', '--side-file={}'.format(tmp_path/"*.json")]):
+        main.main()
```
Comment (rsp9u) [inline tests/unit/test_main.py:48]: Where have the assertions got to?

File: tests/unit/test_main.py [hunk 2]
```diff
@@ -20,3 +22,51 @@ def test_main_with_glob_no_match(mocker):
...
+    sidefile_b = tmp_path / "b.json"
+    orig_test_project_b = {
+        'suites': [{'tests': ['foobar_a', 'foobar_b']}],
+        'tests': [{'id': 'foobar_b', 'commands': []}], 'id': 'foobar_b'
+    }
+    sidefile_b.write_text(json.dumps(orig_test_project_b))
+
+    # call main with mocked driver, etc...
+    with mock.patch.object(config.sys, 'argv', ['prog_name', '--side-file={}'.format(tmp_path/"*.json")]):
+        main.main()
```
Comment (rsp9u) [inline tests/unit/test_main.py:72]: ditto

**Review by rsp9u — CHANGES_REQUESTED** [review]

File: tests/unit/test_main.py (hunk 1 above)
Comment (yaamai) [reply tests/unit/test_main.py:48]: Currently, only checks function does not raise exceptions.

Comment (rsp9u) [reply tests/unit/test_main.py:48]: uh... well.

**Review by rsp9u — APPROVED** [review]

**Pull request merged**